conf/messages.en

```diff
@@ -82,11 +82,11 @@ contact.name.plural.withS = {0}’
 
 contactEmail.title = What is their email address?
 contactEmail.heading = What is the email address for {0}?
-contactEmail.hint = We will use this to send confirmation of your reports or if we have any questions about them.
+contactEmail.hint = We will use this email address to send confirmation of your reports or if we have any questions about them.
```

Comment:
@AashuChauhanHmrc This ticket requires that this is no longer hint text, it should be paragraph text. There should also be changes made to the view to reflect this change